From 2a8b287d3379658c0388f43e49640916105db7c2 Mon Sep 17 00:00:00 2001
From: Robert Ancell <robert.ancell@canonical.com>
Date: Fri, 15 Nov 2019 15:53:48 +1300
Subject: [PATCH] applications: Fix link to privacy panel

The privacy panel was removed in a4bdc37, so this link is broken.
We link to the first panel in the privacy section, which is what
is show when selecting this section.
---
 panels/applications/cc-applications-panel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/applications/cc-applications-panel.c b/panels/applications/cc-applications-panel.c
index 91ec65979..47803085c 100644
--- a/panels/applications/cc-applications-panel.c
+++ b/panels/applications/cc-applications-panel.c
@@ -140,7 +140,7 @@ privacy_link_cb (CcApplicationsPanel *self)
   CcShell *shell = cc_panel_get_shell (CC_PANEL (self));
   g_autoptr(GError) error = NULL;
 
-  if (!cc_shell_set_active_panel_from_id (shell, "privacy", NULL, &error))
+  if (!cc_shell_set_active_panel_from_id (shell, "location", NULL, &error))
     g_warning ("Failed to switch to privacy panel: %s", error->message);
 
   return TRUE;
-- 
2.24.0

